src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -577,8 +577,8 @@ private NetworkPeer(bool inbound, NetworkAddress peerAddress, Network network, N
 
             this.MessageProducer = new MessageProducer<IncomingMessage>();
 
-            this.preferredTransactionOptions = NetworkOptions.All;
-            this.SupportedTransactionOptions = NetworkOptions.None;
+            this.preferredTransactionOptions = network.NetworkOptions;
+            this.SupportedTransactionOptions = network.NetworkOptions & ~NetworkOptions.All;
```

Comment:
Don't we have a `network.NetworkOptions.All` ?

Reply:
The constant does not include the POS flag. This should have the same effect except we include the POS flag from the network.NetworkOptions.